boards/px4/fmu-v2/init/rc.board_sensors

```diff
@@ -14,6 +14,12 @@ hmc5883 -C -T -I -R 4 start
 # Internal SPI bus ICM-20608-G
 mpu6000 -T 20608 start
 
+# External SPI
+ms5611 -S start
```

Comment:
Why external here?

Reply:
Because there was extrnal on the common rc, I didn't want to remove if it possible to have one.
Should it be removed?